raiden/api/rest.py

```diff
@@ -1633,6 +1634,37 @@ def initiate_payment_light(
         # return api_response(result=result.data)
         return None
 
+    def _settle_light(self,
+                      registry_address: typing.PaymentNetworkID,
+                      channel_state: NettingChannelState,
+                      signed_settle_tx: typing.SignedTransaction):
+        """ This operation validates and send the settlement signed transaction for a LC """
+        log.debug(
+            "Settling light channel",
+            node=pex(self.raiden_api.address),
+            registry_address=to_checksum_address(registry_address),
+            channel_identifier=channel_state.identifier
+        )
+
+        self.validate_channel_status_for_settle(channel_state)
+
+        try:
+            self.raiden_api.channel_settle_light(
+                registry_address=registry_address,
+                token_address=channel_state.token_address,
+                creator_address=channel_state.our_state.address,
+                partner_address=channel_state.partner_state.address,
+                signed_settle_tx=signed_settle_tx
+            )
+        except InsufficientFunds as e:
+            return api_error(errors=str(e), status_code=HTTPStatus.PAYMENT_REQUIRED)
+        except RawTransactionFailed as e:
+            return ApiErrorBuilder.build_and_log_error(errors=str(e), status_code=HTTPStatus.BAD_REQUEST, log=log)
+        except RaidenRecoverableError as e:
```

Comment:
```suggestion
        except (RawTransactionFailed, RaidenRecoverableError) as e:
```

Reply:
I extracted this from the current settle function, i will take a look at it